.github/workflows/icu_ci.yml

```diff
@@ -54,12 +60,10 @@ jobs:
       - name: ICU4J
         run: |
           cd icu4j;
-          ant init;
-          ant check;
-          ant localespiCheck
+          mvn install
```

Comment:
Can developers normally do something like `mvn check` or is it always `mvn install` now?

Reply:
There is no `mvn check`. You can run:

-  `mvn test` to only run unit tests
-  `mvn integration-test` or `mvn package` to run unit tests & integration tests

We have info on usage already in https://github.com/unicode-org/icu/blob/main/icu4j/maven-build/README.md. Mihai has more docs on his local machine (not visible in a PR yet) that he is preparing for Step 4 that will go into further detail to allow people to understand why/how about Maven, beyond just the usage.